docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -143,9 +137,9 @@ To add a custom license, add a file with the license text to `custom/options/lic
 
 ### Locales
 
-Locales are managed via our [crowdin](https://crowdin.com/project/gitea).  
-You can override a locale by placing an altered locale file in `custom/options/locale`.  
-Gitea's default locale files can be found in  the [`options/locale`](https://github.com/go-gitea/gitea/tree/master/options/locale) source folder and these should be used as examples for your changes.  
+Locales are managed via our [crowdin](https://crowdin.com/project/gitea).
+You can override a locale by placing an altered locale file in `custom/options/locale`.
+Gitea's default locale files can be found in  the [`options/locale`](https://github.com/go-gitea/gitea/tree/master/options/locale) source folder and these should be used as examples for your changes.
```

Comment:
Whitespace deletion (unrelated changes) here?

Reply:
would say this is ok ... code formation in seperate PR is just noice ...

Reply:
@bagasme my text editor is configured to automatically trim end of line spaces.

Reply:
@bagasme turns out you had a good point. end of line whitespace has its use with markdown which I didn't know about.